PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/springcloud/dependency/action/AddAzureDependencyAction.java

```diff
@@ -75,48 +76,55 @@ public boolean onActionPerformed(@NotNull AnActionEvent event, @Nullable Operati
         final MavenProject mavenProject = projectsManager.findProject(module);
         if (mavenProject == null) {
             PluginUtil.showErrorNotificationProject(project, "Error",
-                                             String.format("Project '%s' is not a maven project.", project.getName()));
+                                                    String.format("Project '%s' is not a maven project.",
+                                                                  project.getName()));
             return true;
         }
+        projectsManager.scheduleImportAndResolve(true);
```

Comment:
how much time will this step cost? Shall we move it to the progress job below?

Reply:
It is very quick, and must be wait here